src/main/kotlin/com/example/travelermileageservice/interfaces/controller/EventController.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.travelermileageservice.interfaces.controller
+
+import com.example.travelermileageservice.interfaces.request.EventCreateRequest
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/events")
+class EventController : MockController {
+
+    @PostMapping
+    override fun saveEvent(request: EventCreateRequest): ResponseEntity<HttpStatus> {
+        return super.saveEvent(request)
+    }
```

Comment:
응답 객체가 없을때는 `ResponseEntity` 를 선언하지 않아도 되요.

```suggestion
    override fun saveEvent(request: EventCreateRequest) {
        super.saveEvent(request)
    }
```

응답의 차이는 ResponseEntity<Unit> 는 empty response body 가 나가고 Unit 은 body 가 나가질 않습니다.